scripts/layers_master.py

```diff
@@ -8,19 +8,39 @@
             "title": "The base Layer class",
             "generate": [
                 "tensorflow.keras.layers.Layer",
-                "tensorflow.keras.layers.Layer.weights",
-                "tensorflow.keras.layers.Layer.trainable_weights",
+                # Properties sorted
```

Comment:
Please sort them 1. semantically, 2. by usage frequency (e.g. pageviews on tensorflow.org or hits in CS).

Semantically would mean grouping related methods together, e.g.

```
weights
trainable_weights
non_trainable_weights
get_weights
set_weights
```

or

```
get_config
from_config
```

By usage would mean listing the most used method groups first.